.github/workflows/test.yml

```diff
@@ -58,15 +58,7 @@ jobs:
           sudo chmod +x "${{ matrix.compose_path }}/docker-compose"
 
       - name: Integration Test
-        run: |
-          echo "Testing initial install"
-          ./install.sh
-          ./_integration-test/run.sh
-          echo "Testing in-place upgrade"
-          # Also test plugin installation here
-          echo "sentry-auth-oidc" >> sentry/requirements.txt
-          ./install.sh --minimize-downtime
-          ./_integration-test/run.sh
+        run: ./integration-test.sh
```

Comment:
The main purpose of factoring this out is to make it easier to run locally, yes? (cf. CONTRIBUTING.md). In that case, how does `./integration-test.sh` relate to `./test.sh`? Looks like there has been some back and forth on this with 004d41bc4d64ff3af7eb970474c04ccb1a703b86. :)

Reply:
> The main purpose of factoring this out is to make it easier to run locally, yes?

Exactly.

> In that case, how does ./integration-test.sh relate to ./test.sh?

I am not sure how to answer this. I think their purpose follows from their contents and usage in the project. Would you like to rename them or comment them?